### PR TITLE
[INLONG-10443][Agent] Put Rocksdb into the plugins module

### LINKED ISSUE #10443: [Improve][Agent] Put Rocksdb into the plugins module
### Description

Place Rocksdb in the plugins module and adjust the relevant unit tests

### InLong Component

InLong Agent

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by justinwwhuang**
Fixes #10443 

### Motivation

Place Rocksdb in the plugins module and adjust the relevant unit tests

### Modifications

1 Place Rocksdb in the plugins module and adjust the relevant unit tests
2 Add storage type configuration instructions

### Verifying this change

*(Please pick either of the following options)*

- [x] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

No doc needed

### CODE REVIEW

File: inlong-agent/conf/agent.properties [hunk 1]
```diff
@@ -99,4 +100,4 @@ agent.prometheus.exporter.port=9080
 # whether to enable audit
 audit.enable=true
 # audit proxy address
-audit.proxys=127.0.0.1:10081
\ No newline at end of file
+audit.proxys.null=
```
Comment (EMsnap) [inline inlong-agent/conf/agent.properties:103]: variable name "audit.proxys.null" may be confusing

Comment (justinwwhuang) [reply inlong-agent/conf/agent.properties]: fix it next issue

File: inlong-agent/conf/agent.properties [hunk 2]
```diff
@@ -18,10 +18,11 @@
 # Agent config values can overwrite the values in the Constants class.
 
 ########################
-# bdb config
+# store config
```
Comment (dockerzhang) [inline inlong-agent/conf/agent.properties:21]:
```suggestion
```

File: inlong-agent/conf/agent.properties [hunk 3]
```diff
@@ -18,10 +18,11 @@
...
+# Offset storage type, optional, defaults to org.apache.inlong.agent.plugin.store.RocksDBStoreImpl
```
Comment (dockerzhang) [inline inlong-agent/conf/agent.properties:22]:
```suggestion
# Offset storage type, default is org.apache.inlong.agent.plugin.store.RocksDBStoreImpl, means store to local RocksDB
```

**Review by aloyszhang — DISMISSED** [review]

File: inlong-agent/conf/agent.properties [hunk 4]
```diff
@@ -100,4 +100,4 @@ agent.prometheus.exporter.port=9080
 audit.enable=true
 # audit proxy address
```
Comment (dockerzhang) [inline inlong-agent/conf/agent.properties:101]:
```suggestion
```

**Review by dockerzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**